.github/workflows/mac_release.yml

```diff
@@ -20,29 +19,40 @@ jobs:
       # fail-fast: Default is true, switch to false to allow one platform to fail and still run others
       fail-fast: false
       matrix:
-        macos_dev_target: [10.15, 11.6, 12.1]
+        macos_dev_target: [10.15, 11.6, 12.1, 13.0]
         include:
         - macos_dev_target: 10.15
           os: macos-11
           allow_failure: false
+          arch: x86_64
+          python: 3.8
         - macos_dev_target: 11.6
           os: macos-11
           allow_failure: false
+          arch: x86_64
+          python: 3.8
         - macos_dev_target: 12.1
           os: macos-12
           allow_failure: false
+          arch: x86_64
+          python: 3.8
+        - macos_dev_target: 13.0
+          os: macos-14
+          allow_failure: false
+          arch: arm64
+          python: 3.12.0
```

Comment:
Why? I find that quite weird that everyone is on 3.8 but mac M1 is on 3.12.0?

Reply:
As discussed offline, this was purely thrown in as a test since GitHub Actions doesn't have 3.8 available for the ARM image.